File: src/main/java/top/zbeboy/isy/web/bean/data/building/BuildingBean.java
package top.zbeboy.isy.web.bean.data.building;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.Building;

/**
 * Created by zbeboy on 2017/5/27.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class BuildingBean extends Building{
    private String schoolName;
    private String collegeName;
}

File: src/main/java/top/zbeboy/isy/web/bean/data/building/BuildingBean.kt
package top.zbeboy.isy.web.bean.data.building

import top.zbeboy.isy.domain.tables.pojos.Building

/**
 * Created by zbeboy 2017-12-07 .
 **/
class BuildingBean: Building() {
    var schoolName: String? = null
    var collegeName: String? = null
    var schoolId: Int = 0
}